src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+export const addBook = (arr) => ({
+  type: 'books/bookAdded',
+  index: 0,
+  payload: {
+    arr,
+  },
+});
+
+export const removeBook = (arr) => ({
+  type: 'books/bookRemoved',
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const ADD_BOOK = bookstore/books/ADD_BOOK